packages/wasm/rollup.config.js

```diff
@@ -1,13 +1,15 @@
-const { builtinModules } = require('module');
+import typescript from '@rollup/plugin-typescript';
 
-const pkg = require('./package.json');
-
-const dependencies = Object.keys(pkg.dependencies || {});
+import pkg from './package.json';
 
 export default [
   {
-    input: 'src/index.js',
-    output: { exports: 'named', file: 'dist/index.js', format: 'cjs' },
-    external: [...builtinModules, ...dependencies]
+    input: 'src/index.ts',
+    plugins: [typescript()],
+    external: ['fs', 'path'],
+    output: [
+      { format: 'cjs', file: pkg.main, exports: 'named' },
```

Comment:
Let's follow the path forward that @lukastaegert laid out here: https://github.com/rollup/plugins/issues/360#issuecomment-622997038

This is also an opportunity to create a "base" rollup config for plugins. Should help reduce duplication across the repo.

Reply:
Should we place a base config in root or next to the test utils?

Reply:
`/shared/rollup.confg.js` would probably work at the root.